package.json

```diff
@@ -37,23 +37,7 @@
     "aiKey": "AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217",
     "main": "./out/extension",
     "activationEvents": [
-        "onCommand:vscode-edge-devtools.attach",
-        "onCommand:vscode-edge-devtools.launch",
-        "onCommand:vscode-edge-devtools.attachToCurrentDebugTarget",
-        "onWebviewPanel:vscode-edge-devtools",
-        "onView:vscode-edge-devtools-view.targets",
-        "onDebug",
-        "onLanguage:css",
-        "onLanguage:html",
-        "onLanguage:javascript",
-        "onLanguage:javascriptreact",
-        "onLanguage:json",
-        "onLanguage:jsonc",
-        "onLanguage:less",
-        "onLanguage:sass",
-        "onLanguage:scss",
-        "onLanguage:typescript",
-        "onLanguage:typescriptreact"
+        "onStartupFinished"

```

Comment:
I'm thinking that we just activate post-VSCode startup.  My thought here is that I want the context menu options available before a user opens an html file.  We already activate on almost all frontend related file types, so the only difference is that we might unnecessarily activate for users who have our extension installed but are working on non-frontend stuff.

Thoughts?

Reply:
Just curious, what does Live Preview do?

Reply:
Live Preview uses this:
```
"onLanguage:html",
"onWebviewPanel:browserPreview",
"onCommand:livePreview.start",
"onCommand:livePreview.start.preview.atFile",
"onCommand:livePreview.start.debugPreview.atFile",
"onCommand:livePreview.start.internalPreview.atFile",
"onCommand:livePreview.start.externalPreview.atFile",
"onCommand:livePreview.start.externalDebugPreview.atFile"
```

However, Live Server uses `onStartupFinished`.

Reply:
I don't personally have a strong preference if both approaches work. My main concern is whether removing the explicit `onLanguage:` entries negatively impacts webhint behavior (as I thought we read these entries from the package.json somewhere in code...)

Reply:
I see.  You mean this part?
https://github.com/microsoft/vscode-edge-devtools/blob/main/src/utils.ts#L825

Yeah, maybe this would be a good time to persist the list of supported file types outside of the activation event section of our package.json.

Reply:
Check out the latest commit and let me know what you think:
https://github.com/microsoft/vscode-edge-devtools/pull/1057/commits/705e1ff4b43ce91ff0edd4782b85365247e4dd6f

Reply:
Hmm... doesn't look like webhint is working for me in the extension with these changes. VS Code pops this error:
![image](https://user-images.githubusercontent.com/785009/172443668-fb5c29f2-6ece-4f5c-9e58-4692f7b1bb9a.png)